input/resources/au-core-medicationrequest.xml

```diff
@@ -247,9 +247,9 @@
       <constraint>
         <key value="au-core-medreq-03"/>
         <severity value="error"/>
-        <human value="Date shall be at least to day"/>
-        <expression value="toString().length() &gt;= 10"/>
-        <xpath value="f:matches(effectiveDateTime,/\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z)/)"/>
+        <human value="Date is precise to the day or, if not available, the Data Absent Reason extension shall be present"/>
+        <expression value="(toString().length() &gt;= 10) xor extension('http://hl7.org/fhir/StructureDefinition/data-absent-reason').exists()"/>
```

Comment:
When I tested, this invariant doesn't didn't fail an example that has a year in the date and an extension. Both a date & DAR extension can be supplied and pass.

Reply:
Noted JIRA https://jira.hl7.org/browse/FHIR-46486 was created.